src/views/ContinuationIn/ContinuationInReviewConfirm.vue

```diff
@@ -96,6 +97,18 @@
         @isFutureEffective="setIsFutureEffective($event)"
       />
     </section>
+    <section
+      v-else
+      id="continuation-effective-date-time-section"
+      class="mt-10"
+    >
+      <header>
+        <h2>Continuation Effective Date and Time</h2>
+        <p class="mt-4">
+          The incorporation date and time will be upon approval.
+        </p>
+      </header>
+    </section>
```

Comment:
New section to show if filing status isn't draft or isn't future effective. In other words, if the filing is Change Requested or Immediate.